### ISSUE TITLE
about grayscale range of data

### ISSUE THREAD

**Issue by weilongMao** [DeweiHu/OCT_DDPM#3]
Is there any requirement for the grayscale range of data?
Do I need to normalize the data to [0,1] ?
Thank you very much!

**Comment by DeweiHu** [comment]
Yes, the image intensity range should be normalized to [0,1]. Otherwise, the Gaussian noise specified in range [-1,1] will not be sufficient for diffusion.

Dewei
________________________________
From: UJSdog ***@***.***>
Sent: Tuesday, November 29, 2022 11:19 AM
To: DeweiHu/OCT_DDPM ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [DeweiHu/OCT_DDPM] about grayscale range of data (Issue #3)


Is there any requirement for the grayscale range of data?
Do I need to normalize the data to [0,1] ?
Thank you very much!

**Comment by eat-orange1** [comment]
Hello, I see that your code normalizes the data to [-1,1], so how do you say normalize to [0,1]

**Comment by DeweiHu** [comment]
Hi, I updated the normalization code earlier this year, it will normalize the intensity to [-1,1] now.

**Comment by eat-orange1** [comment]
Thank you very much for your answer. I am also using normalization to [-1,1]. During the training process, I used a noisy OCT image with a size of 512 * 512, but the loss did not decrease or converge during the training process. Do you know the reason.

**Comment by DeweiHu** [comment]
Hi, I uploaded a checkpoint of my model in the folder ckpts. I normalized my instensity range to [1,3] when training this model. You can try if it works for you.

**Comment by Koid217** [comment]
> 非常感谢您的回答。我也使用规范化来 [-1，1]。在训练过程中，我使用了大小为 512 * 512 的噪声 OCT 图像，但在训练过程中损失没有减少或收敛。你知道原因吗。

Hello, I have such a problem now, do you know the reason？

**Comment by aneetachris** [comment]
i also feel that the denoising is not taking place after the training.
